Scripts/BuildAllPackages.bat

```diff
@@ -0,0 +1,4 @@
+@echo off
+
+cd ..\Packages\
+start %MSFS_SDK%Tools\bin\fspackagetool.exe ".\SimCache.xml"
```

Comment:
I guess one question -- do we want to set the `-nopause` argument (https://docs.flightsimulator.com/html/Additional_Information/Tools/Package_Tool/Package_Tool.htm)?  (Or any of the other ones for that matter?)